packages/react-magma-dom/src/components/Stepper/Stepper.test.js

```diff
@@ -0,0 +1,349 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { axe } from '../../../axe-helper';
+import { magma } from '../../theme/magma';
+import { Stepper, Step, BreakPointStyle } from '.';
+import { fireEvent, render } from '@testing-library/react';
+import { transparentize } from 'polished';
+
+const TEXT = 'Test Text';
+const testId = 'test-id';
+
+describe('Stepper', () => {
+  it('should find element by testId', () => {
+    const { getByTestId } = render(<Stepper testId={testId}>{TEXT}</Stepper>);
+
+    expect(getByTestId(testId)).toBeInTheDocument();
+  });
+
+  it('Does not violate accessibility standards', () => {
+    const { container } = render(<Stepper>{TEXT}</Stepper>);
```

Comment:
This wouldn't be used like this (text inside the Stepper component), so I'm not sure this is a helpful test.

Reply:
🥲